scanner/matcher/matcher.go

```diff
@@ -40,7 +40,11 @@ var (
 		"suse",
 		"ubuntu-matcher",
 	}
-)
+	if env.ScannerV4NodeJSSupport.BooleanSetting() {
```

Comment:
What about the exporter/updater?

Reply:
I guess the PR says "support matching." Let me know if you plan to add vulns to the vulnerability bundle at another time.

Reply:
it's already enabled via `"osv"`